import java.util.*;
class Demo
		{
			public static void main(String args[]) 
				{
				Scanner sc = new Scanner(System.in);
				
					
								try
								{
								ArithmeticException obj = new ArithmeticException();
								throw obj;
								}
								catch(ArithmeticException e)
								{
								System.out.println("\n\n Arith except found, enter +ve no");

								}
								

				
					
					
				}
		



			
						
		}
